### PR TITLE
#349: Mobile import page

### PR DESCRIPTION
**Pull request by ghost**
Update desktop migrate module to mobile
- add expander directive
- update email clients views and controllers

### CODE REVIEW

File: www/modules/migrate/controllers/google-calendar.client.controller.es6 [hunk 1]
```diff
@@ -3,16 +3,17 @@ const DATE_FORMAT = 'yyyy-MM-dd';
```
Comment (ghost) [inline www/modules/migrate/controllers/google-calendar.client.controller.es6]: Is better to store const in https://docs.angularjs.org/api/auto/service/$provide#constant

File: www/modules/migrate/controllers/outlook.client.controller.es6 [hunk 2]
```diff
@@ -1,46 +1,58 @@
 const DATE_FORMAT = 'yyyy-MM-dd';
```
Comment (ghost) [inline www/modules/migrate/controllers/outlook.client.controller.es6]: Is better to store const in https://docs.angularjs.org/api/auto/service/$provide#constant

File: www/modules/migrate/controllers/outlook.client.controller.es6 [hunk 3]
```diff
@@ -1,46 +1,58 @@
...
-            this.authorized = true;
-        } else {
-            this.OutlookService.getAuthUrl().then(data => {
-                this.authUrl = data.url;
-            });
-        }
-    }
+  deleteClientCookies() {
+    delete this.$cookies['outlook_access_token'];
+    delete this.$cookies['outlook_email'];
```
Comment (ghost) [inline www/modules/migrate/controllers/outlook.client.controller.es6]: Put constant string to https://docs.angularjs.org/api/auto/service/$provide#constant

File: www/modules/migrate/controllers/outlook.client.controller.es6 [hunk 4]
```diff
@@ -1,46 +1,58 @@
...
-            }
-        });
+    this.initialize();
+  }
 
-        this.initialize();
+  initialize() {
+    if (this.$cookies['outlook_access_token'] && this.$cookies['outlook_email']) {
+      this.params.token   = this.$cookies['outlook_access_token'];
+      this.params.email   = this.$cookies['outlook_email'];
```
Comment (ghost) [inline www/modules/migrate/controllers/outlook.client.controller.es6]: Put constant string to https://docs.angularjs.org/api/auto/service/$provide#constant

File: www/modules/migrate/services/google-calendar.client.service.es6 [hunk 5]
```diff
@@ -1,33 +1,33 @@
...
+  convertCalendarEvents(events) {
+    return events.map(event => {
+      return {
+        id:    event.id,
+        type:  'event',
+        title: event.summary,
+        notes: event.description,
+        days:  {
+          startTime: event.start.dateTime,
+          endTime:   event.end.dateTime
```
Comment (lotrien) [inline www/modules/migrate/services/google-calendar.client.service.es6:26]: Could you please double check whether it's `dateTime` or just `time` comes from google-calendar API

**Pull request merged**